_i18n/en/general/privacy.md

```diff
@@ -1,57 +1,82 @@
-- Google: If you have activated backup & reset in your phone settings (`Settings` » `Backup & Reset` » `Back up my data`), you should be aware that Android itself will periodically save a copy of your phone's data in Google's servers. This backup contains private information, including your WiFi passwords, messages and call history. It may also include data from AntennaPod and from other apps you use. The developers of AntennaPod do not have access to this data. For more information, see [Google's privacy policy](https://policies.google.com).
+#### Podcast hosters
+- When provided in the podcast settings, a host will receive authentication data you provided.
+- Web servers that provide podcast feeds may collect additional data, such as your IP address, access time and what is being accessed. This includes the episodes you are downloading or streaming. Please refer to their respective privacy policies for details. You can view a podcast’s URL by opening the podcast and pressing the info icon. AntennaPod does not allow remote servers to set cookies. If multiple podcasts are hosted on the same server, the server can detect the list of podcasts that you subscribed to and that are hosted on that server. This can happen when publishers use services like FeedBurner, Podtrac, or similar to distribute their feeds.
+- Servers can detect that you are using AntennaPod and which version (via the HTTP User-Agent).
 
-## What data might be collected, stored and processed from AntennaPod online services
-- The donation system: AntennaPod uses Open Collective, offered by Open Collective Inc (USA) & Open Collective Europe ASBL (Belgium). When you make a donation, these entities get access to certain data. For more information, see [Open Collective's privacy policy](https://opencollective.com/privacypolicy).
-- The website: AntennaPod's website is hosted via GitHub Pages. [GitHub's privacy policy](https://docs.github.com/en/github/site-policy/github-privacy-statement#github-pages) notices that "GitHub may collect User Personal Information from visitors to your GitHub Pages website, including logs of visitor IP addresses, to comply with legal obligations, and to maintain the security and integrity of the Website and the Service." The website does not store any cookies on your machine, and no 3rd party tracking (analytics) or other services are used.
-- The forum: AntennaPod's forum uses the Discourse software, hosted by the core team. Please see the [forum's privacy policy](https://forum.antennapod.org/privacy) for which data is concerned.
+#### Discovery and search services
+- When opening the Discover screen in the app, it sends a call to Apple to collect podcast suggestions. They may store the query, including country (which by default is the device's country). These suggestions can be disabled by choosing "off" in the region selector (already the default for the release on F-Droid).
```

Comment:
"by choosing "off" in the region selector" -> no, by selecting the checkbox "hide"

"already the default for the release on F-Droid" -> no, that's just a confirmation dialog in front, not the "hide" checkbox

Reply:
> "already the default for the release on F-Droid" -> no, that's just a confirmation dialog in front, not the "hide" checkbox

Good to know it's technically different, but the result is the same (no calls made to Apple), right?

I'll have to think about the phrasing then.